ambry-admin/src/main/java/com.github.ambry.admin/AdminSecurityService.java

```diff
@@ -98,14 +98,14 @@ public AdminSecurityService(AdminConfig adminConfig, AdminMetrics adminMetrics)
             RestUtils.SubResource subResource = RestUtils.getBlobSubResource(restRequest);
             if (subResource == null) {
               Long ifModifiedSinceMs = getIfModifiedSinceMs(restRequest);
+              responseChannel.setHeader(RestUtils.Headers.LAST_MODIFIED,
```

Comment:
Yeah looked at this too - but here it isn't possible since there is another subresource called `Replicas` for which `Last-Modified` does not make sense.